src/SparkTestBase.sol

```diff
@@ -231,9 +392,20 @@ abstract contract SparklendTestBase is ProtocolV3TestBase, SparkTestBase {
         vm.prank(admin);
         return InitializableAdminUpgradeabilityProxy(payable(proxy)).implementation();
     }
+
+    function _getPoolAddressesProviderRegistry() internal view returns(IPoolAddressesProviderRegistry registry ){
```

Comment:
I feel this could be made into part of the `setupDomains` functions but ran out of ideas. Will try again tomorrow, but feedback/ideas are of course welcome

Reply:
kept the function as syntax sugar, moved choosing the actual address to `setupDomains`